### DISCUSSION TITLE
SvelteKit build errors

### DISCUSSION THREAD

**Discussion by samfcheng** [vercel/community#1459] [Help] [answered]
### Summary

Hello!
Both me and someone else in the Svelte discord are encountering the same problem. Building/deploying a SvelteKit app to Vercel has been completely fine, but recently started creating errors in the same projects.

![image](https://user-images.githubusercontent.com/51756184/217146061-eb11e84a-21ca-47ec-abb9-4a2c4452be10.png)

### Additional information

Yesterday:
![image](https://user-images.githubusercontent.com/51756184/217146082-e18cabe5-fcb7-4e53-b95c-fd968b4052a6.png)

Today:
![image](https://user-images.githubusercontent.com/51756184/217146092-1f98d9a7-790f-4cdd-81d8-28796fdc35d1.png)

![image](https://user-images.githubusercontent.com/51756184/217146235-c4a7f19c-1937-4482-8762-e89c9042c62f.png)

**Comment by swarnava** [comment, marked as answer]
Looks like Sveltjs adapter has been released recently recently which has been introduced some breaking changes: https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fadapter-vercel%402.0.0

I will recommend filling a Issue on the GitHub of respective repository:https://github.com/sveltejs/kit/issues
